.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,84 @@
+name: Generate Lighthouse report
+
+on:
+  issue_comment:
+    types: [edited]
+
+jobs:
+  generate_lighthouse_report:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout to repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+          ref: master
+
+      - name: Add Lighthouse progress comment
+        id: generate_lighthouse_comment
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Ensure that `GITHUB_TOKEN` access for actions have read and write access, otherwise this error will occur:
<img width="606" alt="Screenshot 2023-01-06 at 2 44 37 PM" src="https://user-images.githubusercontent.com/103016120/210945590-577df243-068e-471b-8797-b065ba0d2b1f.png">

Reply:
However this error is happening only on my side because for forked repositories, read-only access is provided for Github actions

Reply:
In `deriv-com` and our `generate_app_id` workflow, we also use `GITHUB_TOKEN` so this should be fine once merged and tested 🤌